pkg/virt-controller/services/template.go

```diff
@@ -763,19 +763,10 @@ func (t *templateService) RenderHotplugAttachmentPodTemplate(volumes []*v1.Volum
 		Spec: k8sv1.PodSpec{
 			Containers: []k8sv1.Container{
 				{
-					Name:    hotplugDisk,
-					Image:   t.launcherImage,
-					Command: command,
-					Resources: k8sv1.ResourceRequirements{ //Took the request and limits from containerDisk init container.
-						Limits: map[k8sv1.ResourceName]resource.Quantity{
-							k8sv1.ResourceCPU:    resource.MustParse("100m"),
-							k8sv1.ResourceMemory: resource.MustParse("80M"),
-						},
-						Requests: map[k8sv1.ResourceName]resource.Quantity{
-							k8sv1.ResourceCPU:    resource.MustParse("10m"),
-							k8sv1.ResourceMemory: resource.MustParse("2M"),
-						},
-					},
+					Name:      hotplugDisk,
+					Image:     t.launcherImage,
+					Command:   command,
+					Resources: initContainerResourceRequirementsForVMI(vmi),
```

Comment:
This might be confusing so I would leave a comment. Also looking into the function I would expect that the limits for not dedicated VM would match limits with dedicated VM.

Reply:
I forget the details, but some time ago we had an issue with container disks running out of memory in some very specific situation. And that is why the limit was changed to 80M instead of 40M. I believe that is the reason why the limit on non-dedicated is higher. But not sure of the exact details.

Reply:
Yeah, and it's been like this for ages apparently (0.36):
https://github.com/kubevirt/kubevirt/blob/df9740d63ea3125388859131afd66c4af5f23f82/pkg/virt-controller/services/template.go#L1104-L1117